src/util.ts

```diff
@@ -279,21 +283,16 @@ export function getManifestsFromRegistry(): Promise<string[]> {
   });
 }
 
-function removeManifestFromRegistry(manifestPath: string): Promise<any> {
+async function removeManifestFromRegistry(manifestPath: string): Promise<any> {
   if (!manifestPath) {
     return new Promise((resolve, reject) => {
```

Comment:
If this is an async method, this code can now just `throw new Error("No manifest was specified.");` without having to create a promise.